src/cudamatrix/cu-array-test.cc

```diff
@@ -73,6 +73,8 @@ static void UnitTestCuArray() {
       CuArray<T> cu_vec(vec);
       std::vector<T> vec2;
       cu_vec.CopyToVec(&vec2);
+      T *vec22 = new T[vec.size()];
+      cu_vec.CopyToHost(vec22);
     }
```

Comment:
Please fix memory leak in test code.  (we might run valgrind on it at some point to check for memory leaks).